### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by sean-smith**
New format using the github issues template builder. See https://help.github.com/en/articles/about-issue-and-pull-request-templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+**Bug description and how to reproduce:**
+A clear and concise description of what the bug is and the steps to reproduce the behavior.
+
+**Additional context:**
+Any other context about the problem. E.g.:
+ - configuration file without any credentials or personal data.
+ - pre/post-install scripts, if any
+ - screenshots, if useful
+ - if the cluster fails creation, please re-execute `create` action using `--norollback` option and attach `/var/log/cfn-init.log`, `/var/log/cloud-init.log` and `/var/log/cloud-init-output.log` files from the Master node
+ - if you encounter scaling problems please attach `/var/log/nodewatcher` from the Compute node and `/var/log/jobwatcher` and `/var/log/sqswatcher` from the Master node
```
Comment (enrico-usai) [inline .github/ISSUE_TEMPLATE/bug_report.md]: we could add the folder on which we are collecting the nodewatcher logs:  /home/logs/compute

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+name: Bug report
+about: Please create a detailed report by completing the following information
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Environment:**
+ - AWS ParallelCluster / CfnCluster version [e.g. aws-parallelcluster-2.0.0]
```
Comment (enrico-usai) [inline .github/ISSUE_TEMPLATE/bug_report.md]: we could put 2.4.1 as default

Comment (sean-smith) [reply .github/ISSUE_TEMPLATE/bug_report.md]: done

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (sean-smith) [reply .github/ISSUE_TEMPLATE/bug_report.md]: done

**Pull request merged**